converter.py

```diff
@@ -0,0 +1,29 @@
+import os
+import hashlib
+from os import walk
+import mimetypes
+import json
+import shutil
+
+
+class Converter:
+    def __init__(self):
+        self.meta = {}
+
+    def convert(self, path):
+        f = []
+
+        for (dirpath, dirnames, filenames) in walk(path):
+            for fn in filenames:
+                f.append(os.path.join(dirpath, fn))
+        for fn in f:
+            path_len = len(path)
+            file_name = fn[path_len:]
```

Comment:
Previously in line #L18, we can store only `filesnames` and here create a variable `full_file_name` or `full_path` by joining `file_name` and `path`. This method looks cleaner just the same you did in `sanre.py`

Reply:
We can't store just `filenames`. Because we need full path like `/foo/bar/page.html` for hashing, not only `page`. `path` contains path like `/opt/snare/pages/<page>` and we need to ignore it.

Reply:
Isn't `/foo/bar` = `/opt/snare/pages/example.com` = path? I guess that's why you're using `path_len` to determine `file_name` because `fn = path + file_name`.

Reply:
No, this solution  based on `walk` result. Maybe there is a better solution :)

```
>>> import os
>>> for (dirpath, dirnames, filenames) in os.walk('/opt/snare/pages/example.com'):
...     for fn in filenames:
...             print(os.path.join(dirpath, fn))
...
/opt/snare/pages/example.com/page.html
/opt/snare/pages/example.com/foo/page.html
```

So we need `/foo/page.html` for hashing, not `/opt/snare/pages/example.com/foo/page.html` or `page.html`.  

`walk` by iteration
```
 for (dirpath, dirnames, filenames) in os.walk('/opt/snare/pages/example.com'):
...     print("dirpath: {}, dirnames: {}, filenames: {}".format(dirpath, dirnames, filenames))
...
dirpath: /opt/snare/pages/example.com, dirnames: ['foo'], filenames: ['page.html']
dirpath: /opt/snare/pages/example.com/foo, dirnames: [], filenames: ['page.html']
```

Reply:
I didn't consider old style has directories too, my bad.